packages/error-reporting/README.md

```diff
@@ -16,11 +16,12 @@ applications running in almost any environment. Here's an introductory video:
 ## Prerequisites
 
 1. Your application needs to use Node.js version 4.x or greater.
-1. You need a [Google Cloud project](https://console.cloud.google.com). Your application can run anywhere, but errors are reported to a particular project.
-1. [Enable the Stackdriver Error Reporting API](https://console.cloud.google.com/apis/api/clouderrorreporting.googleapis.com/overview) for your project.
-1. The module will only send errors when the `NODE_ENV` environment variable is
+2. You need a [Google Cloud project](https://console.cloud.google.com). Your application can run anywhere, but errors are reported to a particular project.
```

Comment:
Keep the 1's to allow easier updates to the markdown in the future. This is valid (and preferred) way to do numbered lists in markdown. Elsewhere in the file too.